pkg/slo-controller/config/config.go

```diff
@@ -121,3 +135,35 @@ func GetNodeColocationStrategy(cfg *ColocationCfg, node *corev1.Node) *Colocatio
 	}
 	return strategy
 }
+
+// DefaultNodeSLOSpecConfig defines the default config of the nodeSLOSpec, which would be used by the resmgr
+func DefaultNodeSLOSpecConfig() slov1alpha1.NodeSLOSpec {
+	return slov1alpha1.NodeSLOSpec{
+		ResourceUsedThresholdWithBE: DefaultResourceThresholdStrategy(),
+	}
+}
+
+func DefaultResourceThresholdStrategy() *slov1alpha1.ResourceThresholdStrategy {
+	return &slov1alpha1.ResourceThresholdStrategy{
+		Enable:                      pointer.BoolPtr(false),
+		CPUSuppressThresholdPercent: pointer.Int64Ptr(65),
+		CPUSuppressPolicy:           slov1alpha1.CPUSetPolicy,
+	}
+}
+
+// MergeCfg merges the old cfg with the new one, where new's values take precedence over old's when both fields exist.
+// It would throw an error if:
+//   1. either of the inputs was nil;
+//   2. the inputs were not a pointer of the same json struct.
+func MergeCfg(old, new interface{}) (interface{}, error) {
```

Comment:
this function has been implemented in `pkg/util/utils.go`

Reply:
done